dev-docs/bidders/ozone.md

```diff
@@ -15,6 +15,7 @@ usp_supported: true
 floors_supported: true
 prebid_member: true
 sidebarType: 1
+supported Aliases: lmc, venatus
```

Comment:
"supported Aliases" is not a recognized header. The way this needs to be done is to create new files for lmc and venatus, pointing them at ozone. Take a look at how others (e.g. adtelligent) has implemented the docs for aliases.